README.md

```diff
@@ -0,0 +1,20 @@
+# Repository Tools
+This directory contains tools useful when operating on projects with multiple Git repositories on Azure Repos.
+These scripts require Powershell Core.
+
+## clone
+This scripts automates cloning of all repositories within a project. Usage:
+
+`.\Clone-Repos.ps1 -project -pat <devops_personal_access_token>`
```

Comment:
will it work with -project without actually providing project?

Reply:
And I guess the organisation is a required parameter too..., right ?

Reply:
Good catch. I updated it.